packages/ember/src/components/nav-item.gts

```diff
@@ -0,0 +1,32 @@
+import Component from '@glimmer/component';
+import { inject as service } from '@ember/service';
+import ResponsiveService from '../services/responsive.ts';
+
+export interface NrgNavItemSignature {
+  Blocks: {
+    default: [];
+  };
+  Element: HTMLLIElement;
+  Args: {
+    label: string;
+    url: string;
+  };
+}
+
+export default class NrgNavItem extends Component<NrgNavItemSignature> {
+  @service declare responsive: ResponsiveService;
+
+  get classList() {
+    const classes = ['nav-item'];
+    if (this.responsive.isMobileDevice) {
+      classes.push('border-bottom');
+    }
+    return classes.join(' ');
+  }
+
+  <template>
+    <li class={{this.classList}}>
+      <a class="nav-link" href={{@url}}>{{@label}}</a>
```

Comment:
Instead of requiring a URL, can the user also pass a route name (and model)? This could use the `LinkTo` component internally.